recipes/fast-cdr/all/conanfile.py

```diff
@@ -30,33 +28,23 @@ class FastCDRConan(ConanFile):
         "fPIC": True,
     }
 
-    def export_sources(self):
-        export_conandata_patches(self)
-
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
-
-    def configure(self):
-        if self.options.shared:
-            self.options.rm_safe("fPIC")
+    implements = ["auto_shared_fpic"]
 
     def layout(self):
         cmake_layout(self, src_folder="src")
 
     def build_requirements(self):
-        if Version(self.version) >= "1.1.0":
-            self.tool_requires("cmake/[>=3.16.3 <4]")
+        self.tool_requires("cmake/[>=3.16.3]")
 
     def validate(self):
-        if self.settings.compiler.get_safe("cppstd"):
-            check_min_cppstd(self, 11)
+        check_min_cppstd(self, 11)
+
```

Comment:
I don't know if this was an oversight but this MR removed a check for the presence of `cppstd`.
This now raises an error when `cppstd` is not defined in the profile used for a build.

Wouldn't it be better to keep the check and issue a warning in case `cppstd` is not defined?